terraform/app/variables.tf

```diff
@@ -120,90 +104,44 @@ variable "rails_master_key_path" {
   nullable    = false
 }
 
-variable "docker_image" {
-  type        = string
-  default     = "mavis/webapp"
-  description = "The docker image name for the essential container in the task definition"
-  nullable    = false
-}
-
-variable "image_digest" {
-  type        = string
-  description = "The docker image digest for the essential container in the task definition."
-  nullable    = false
-}
-
-variable "enable_cis2" {
-  type        = bool
-  default     = true
-  description = "Boolean toggle to determine whether the CIS2 feature should be enabled."
-  nullable    = false
-}
-
-variable "enable_pds_enqueue_bulk_updates" {
-  type        = bool
-  default     = true
-  description = "Whether PDS jobs that update patients in bulk should execute or not. This is disabled in non-production environments to avoid making unnecessary requests to PDS."
-  nullable    = false
-}
-
-variable "academic_year_today_override" {
-  type        = string
-  default     = "nil"
-  description = "A date that can be used to override today's date when calculating the current academic year."
-  nullable    = false
-}
-
-variable "academic_year_number_of_preparation_days" {
-  type        = number
-  default     = 31
-  description = "How many days before the start of the academic year to start the preparation period."
-  nullable    = false
-}
-
-variable "pds_rate_limit_per_second" {
-  type        = number
-  default     = 5
-  description = "The rate limit when communicating with PDS."
-  nullable    = false
-}
-
-variable "enable_splunk" {
-  type        = bool
-  default     = true
-  description = "Boolean toggle to determine whether the Splunk feature should be enabled."
-  nullable    = false
-}
-
 variable "enable_enhanced_db_monitoring" {
   type        = bool
   default     = false
   description = "Boolean toggle to determine whether enhanced DB monitoring should be enabled."
   nullable    = false
 }
 
-variable "app_version" {
-  type        = string
-  description = "The version identifier for the MAVIS application deployment"
-  default     = "Unknown"
-  nullable    = false
-}
-
 locals {
   is_production = var.environment == "production"
   parameter_store_variables = tomap({
```

Comment:
I think the explicit casts to map are not necessary, also in the other places further down